### PR TITLE
Dependecies version accessible from gui [11745]

### PR DESCRIPTION
**Pull request by rsanchez15**

### CODE REVIEW

File: fastdds_monitor.pro [hunk 1]
```diff
@@ -88,3 +88,11 @@ else:unix: LIBS += -L$$PWD/../../install/fastdds-statistics-backend/lib/ -lfastd
 
 INCLUDEPATH += $$PWD/../../install/fastdds-statistics-backend/include
 DEPENDPATH += $$PWD/../../install/fastdds-statistics-backend/include
+
```
Comment (jparisu) [inline fastdds_monitor.pro:91]: Check in line above <81,82> if `/include` is needed

File: src/Controller.cpp [hunk 2]
```diff
@@ -112,3 +118,38 @@ void Controller::add_statistics_data(
...
+}
+
+QString Controller::fastdds_statistics_backend_version()
+{
+    return utils::to_QString(FASTDDS_STATISTICS_BACKEND_VERSION_STR);
+}
+
+QString Controller::qt_version()
+{
+    return QT_VERSION_STR;
```
Comment (jparisu) [inline src/Controller.cpp:134]: Why not `utils::to_QString`? It looks like a char * as the other macros

Comment (jparisu) [inline src/Controller.cpp:134]: It does not require an include? `qconfig.h`

File: src/Controller.cpp [hunk 3]
```diff
@@ -13,7 +13,13 @@
 // limitations under the License.
 
 #include <QDebug>
+#include <QStringBuilder>
 
+#include <fastrtps/config.h>
+#include <fastdds-statistics-backend/config.h>
+#include <fastdds-monitor/config.h>
+
+#include <fastdds-monitor/utils.h>
```
Comment (jparisu) [inline src/Controller.cpp:22]: Alphabetic order

**Review by jparisu — CHANGES_REQUESTED** [review]
It does not compile:
It does not find `config.h`

File: src/Controller.cpp (hunk 2 above)
Comment (rsanchez15) [reply src/Controller.cpp:134]: ` #include <QtGlobal> `

File: fastdds_monitor.pro (hunk 1 above)
Comment (rsanchez15) [reply fastdds_monitor.pro:91]: Addressed in 1fe2bed

File: fastdds_monitor.pro [hunk 4]
```diff
@@ -96,3 +96,7 @@ else:unix: LIBS += -L$$PWD/../../install/fastrtps/lib/ -lfastrtps
 
 INCLUDEPATH += $$PWD/../../install/fastrtps/include
 DEPENDPATH += $$PWD/../../install/fastrtps/include
+
+# Link to the Fast DDS Monitor includes
```
Comment (jparisu) [inline fastdds_monitor.pro:100]: Should not this replace lines 81 and 82?

**Review by jparisu — APPROVED** [review]
LGTM

NIT .pro file extra lines

File: fastdds_monitor.pro (hunk 4 above)
Comment (rsanchez15) [reply fastdds_monitor.pro:100]: These are the fastdds includes. The others are the fastdds monitor ones

Comment (rsanchez15) [reply fastdds_monitor.pro:100]: These are the fastdds includes. The others are the fastdds monitor ones

**Pull request merged**